pkgs/top-level/all-packages.nix

```diff
@@ -11714,10 +11714,7 @@ with pkgs;
     kernelPatches =
       [ kernelPatches.bridge_stp_helper
         kernelPatches.p9_fixes
-        # See pkgs/os-specific/linux/kernel/cpu-cgroup-v2-patches/README.md
-        # when adding a new linux version
```

Comment:
I think the lines about updating should be kept (looking how we blindly copypasted 4.10 it's better to have some reminder…), but the short-term FIXME below should be deleted